src/server/util/index.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const crypto = require("crypto");
+
+module.exports.getKey = (size) => {
```

Comment:
Really? I figured this was a synchronous function and functions should expect an immediate token generation. Would there be an asynchronous application to this?

Reply:
the token is generated by reading from the operating system's random pool, which is actually an asynchronous function. This just pretends to be synchronous for simplicity. The reason would be for performance reasons, basically while javascript is doing anything synchronous nothing else can be done. The if you make it asynchronous, then it can. Like i said its really nitpicky